geo/src/algorithm/affine_ops.rs

```diff
@@ -141,8 +201,8 @@ impl<T: CoordNum> AffineTransform<T> {
     /// xoff = origin.x - (origin.x * xfact)
     /// yoff = origin.y - (origin.y * yfact)
     /// ```
-    pub fn scale(xfact: T, yfact: T, origin: Point<T>) -> Self {
-        let (x0, y0) = origin.x_y();
+    pub fn scale(xfact: T, yfact: T, origin: impl Into<Coordinate<T>>) -> Self {
```

Comment:
I'm not sure if this is controversial - but otherwise this is always kind of a papercut for me at the call site.

Reply:
IIRC using `Into` in function signatures _can_ lead to less informative error messages but I can't find the ref now and I would hope that the signature makes it obvious what's going on, so 👍

Reply:
Sometimes I daydream about getting rid of one of Point/Coord... 🔥